content/docs/authorization/v2.0 Tech Preview/configuration/proxy-server/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Proxy Server
+linktitle: Proxy Server
+description: >
+  Configuring the CSM for Authorization Proxy Server
+---
+
+## Configuring the CSM for Authorization Proxy Server
+
+Run `kubectl -n authorization get ingress` and `kubectl -n authorization get service` to see the Ingress rules for these services and the exposed port for accessing these services via the LoadBalancer. For example:
+
+```bash
+kubectl -n authorization get ingress
+```
+```
+NAME              CLASS   HOSTS                           ADDRESS   PORTS     AGE
+proxy-server      nginx   csm-authorization.com                     00, 000   86s
+```
+```bash
+kubectl -n authorization get service
+```
+```
+NAME                                               TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)                      AGE
+authorization-cert-manager                         ClusterIP      00.000.000.000    <none>        000/TCP                     28s
+authorization-cert-manager-webhook                 ClusterIP      00.000.000.000    <none>        000/TCP                     27s
+authorization-ingress-nginx-controller             LoadBalancer   00.000.000.000    <pending>     00:00000/TCP,000:00000/TCP  27s
+authorization-ingress-nginx-controller-admission   ClusterIP      00.000.000.000    <none>        000/TCP                     27s
+proxy-server                                       ClusterIP      00.000.000.000    <none>        000/TCP                     28s
+redis-csm                                          ClusterIP      00.000.000.000    <none>        000/TCP                     28s
+rediscommander                                     ClusterIP      00.000.000.000    <none>        000/TCP                     27s
+role-service                                       ClusterIP      00.000.000.000    <none>        000/TCP                     27s
+sentinel                                           ClusterIP      00.000.000.000    <none>        000/TCP                     27s
+storage-service                                    ClusterIP      00.000.000.000    <none>        000/TCP                     27s
+tenant-service                                     ClusterIP      00.000.000.000    <none>        000/TCP                     28s
+```
+
+On the machine running `dellctl`, the `/etc/hosts` file needs to be updated with the Ingress hosts for the proxy, storage, and role services. For example:
```

Comment:
We have to update `/etc/hosts` if a hostname like `csm-authorization.com`, or any hostname that doesn't resolve, is used. If the user configures a hostname that resolves in their environment, they don't have to do this.

We should mention that this is only required if they kept the default, placeholder hostname.